src/test/java/com/google/api/codegen/testdata/python_main_library.baseline

```diff
@@ -839,3 +842,37 @@ class LibraryServiceApi(object):
             name=name)
         return self._get_book_from_archive(request, options)
 
+    def update_book_index(
+            self,
+            name,
+            index_name,
+            index_map,
+            options=None):
+        """
+        Updates the index of a book.
+
+        Example:
+          >>> from library.library_service_api import LibraryServiceApi
+          >>> api = LibraryServiceApi()
+          >>> name = api.book_path('[SHELF]', '[BOOK]')
+          >>> index_name = ''
+          >>> index_map_item = ''
+          >>> index_map = {'default_key', index_map_item,}
```

Comment:
shouldn't there be a colon between key & value in python?

`index_map = {'default_key': index_map_item}`